### PR TITLE
Fix create template modal

### PR DESCRIPTION
**Pull request by negreirosleo**
This PR fix the typescript error in the SaveTemplate form.

[Screencast from 2024-02-19 15-24-59.webm](https://github.com/Igalia/phpreport/assets/20492786/af1ad60e-39c4-4a96-af88-bbeec9aff65c)

### CODE REVIEW

File: frontend/src/app/tasks/components/SaveTemplateModal.tsx [hunk 1]
```diff
@@ -34,6 +34,16 @@ export const SaveTemplateModal = ({ task }: SaveAsTemplateProps) => {
         <Sheet
           variant="outlined"
           component="form"
+          action={async (formData: FormData) => {
+            console.log('test')
```
Comment (anarute) [inline frontend/src/app/tasks/components/SaveTemplateModal.tsx:38]: nit :)

```suggestion
```

**Review by anarute — APPROVED** [review]
LGTM

File: frontend/src/app/tasks/components/SaveTemplateModal.tsx (hunk 1 above)
Comment (negreirosleo) [reply frontend/src/app/tasks/components/SaveTemplateModal.tsx]: you saw nothing :eyes:

**Review by dmtrek14 — APPROVED** [review]
LGTM. Have you tried running `npm run build` to make sure that this will now work during build?  I had changed this initially in order to solve the build errors related to the useFormStatus.

**Comment by negreirosleo** [comment]
@dmtrek14 the build is working! (I forgot to add the @types/react update on the commit earlier)

**Comment by negreirosleo** [comment]
image

Building :pray:

**Pull request merged**